[2022-08-23 20:49:05,568] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: gmailPipeline.gmailAuthTask manual__2022-08-24T00:49:03.287103+00:00 [queued]>
[2022-08-23 20:49:05,573] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: gmailPipeline.gmailAuthTask manual__2022-08-24T00:49:03.287103+00:00 [queued]>
[2022-08-23 20:49:05,573] {taskinstance.py:1376} INFO - 
--------------------------------------------------------------------------------
[2022-08-23 20:49:05,573] {taskinstance.py:1377} INFO - Starting attempt 1 of 1
[2022-08-23 20:49:05,573] {taskinstance.py:1378} INFO - 
--------------------------------------------------------------------------------
[2022-08-23 20:49:05,583] {taskinstance.py:1397} INFO - Executing <Task(PythonOperator): gmailAuthTask> on 2022-08-24 00:49:03.287103+00:00
[2022-08-23 20:49:05,585] {standard_task_runner.py:52} INFO - Started process 6324 to run task
[2022-08-23 20:49:05,592] {standard_task_runner.py:79} INFO - Running: ['airflow', 'tasks', 'run', 'gmailPipeline', 'gmailAuthTask', 'manual__2022-08-24T00:49:03.287103+00:00', '--job-id', '11', '--raw', '--subdir', 'DAGS_FOLDER/gmailDag.py', '--cfg-path', '/var/folders/mm/xv24gcjj46l_jx1m76p5bgy80000gn/T/tmp2h0c7mrv', '--error-file', '/var/folders/mm/xv24gcjj46l_jx1m76p5bgy80000gn/T/tmp17vuuy4u']
[2022-08-23 20:49:05,594] {standard_task_runner.py:80} INFO - Job 11: Subtask gmailAuthTask
[2022-08-23 20:49:05,641] {task_command.py:371} INFO - Running <TaskInstance: gmailPipeline.gmailAuthTask manual__2022-08-24T00:49:03.287103+00:00 [running]> on host abhays-mbp.lan
[2022-08-23 20:49:05,685] {taskinstance.py:1589} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=gmailPipeline
AIRFLOW_CTX_TASK_ID=gmailAuthTask
AIRFLOW_CTX_EXECUTION_DATE=2022-08-24T00:49:03.287103+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2022-08-24T00:49:03.287103+00:00
[2022-08-23 20:49:05,817] {local_task_job.py:156} INFO - Task exited with return code Negsignal.SIGABRT
[2022-08-23 20:49:05,834] {taskinstance.py:1415} INFO - Marking task as FAILED. dag_id=gmailPipeline, task_id=gmailAuthTask, execution_date=20220824T004903, start_date=20220824T004905, end_date=20220824T004905
[2022-08-23 20:49:05,860] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
